src/components/organisms/OrchestrationTable/index.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from 'react'
+import axios from 'axios'
+import './index.css'
+import CheckBox from '../../Atoms/Checkbox'
+import { Box, IconButton, Typography } from '@mui/material'
+import { KeyboardArrowDown } from '@mui/icons-material'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+
+const OrchestrationTable = () => {
+  const [tableData, settableData] = useState<any>()
+  useEffect(() => {
+    axios
+      .get('http://localhost:8080/table_data')
+      .then((res) => settableData(res.data))
+      .catch((err) => console.log(err))
+  }, [])
+
+  const handleCheckBox = (id: number) => {
+    tableData?.data.forEach((data: any) => {
+      if (data.id == id) {
+        data.isChecked = !data.isChecked
+      }
+    })
+    const tableDataUpdated = {
+      headers: tableData?.headers,
+      data: tableData?.data,
+    }
+    settableData({ ...tableDataUpdated })
+  }
+  return (
+    <TableContainer>
+      <Table className="table" aria-label="simple table">
+        <TableHead>
+          <TableRow className="row-head">
+            <TableCell align="center" width={2} className="cell-head">
+              <CheckBox sx={{ transform: 'scale(0.8)', color: '#c1c1c1' }} />
+            </TableCell>
+            {tableData &&
+              tableData.headers.map((data: any) => {
+                if (data != 'Description') {
+                  return (
+                    <TableCell className="cell-head">
```

Comment:
Create a TableCell Atom with Typography component and Styles.
In table just pass props and content.